ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp

```diff
@@ -975,7 +976,7 @@ std::vector<Tensor> _sinh_bw(const Tensor& grad, const Tensor& input, const Memo
         where(
             ttnn::lt(input, ttnn::operations::creation::full_like(input, -88.5), std::nullopt, output_mem_config),
             t_inf,
-            ttnn::multiply(grad, cosh(input, output_mem_config), std::nullopt, output_mem_config),
+            ttnn::multiply(grad, unary::_cosh(input, output_mem_config), std::nullopt, output_mem_config),
```

Comment:
We should use registered ttnn:: operations.
Thats important, because else we won't be able to reconstruct a graph of composite calls later. And thats important for the future roadmap.